dali_tf_plugin/dali_dataset_op.cc

```diff
@@ -295,7 +307,13 @@ class DALIDatasetOp : public DatasetOpKernel {
                   "for tensor " + std::to_string(out_id));
             }
 
-            DALI_CALL(daliCopyTensorNTo(&pipeline_handle_, dst, out_id, device_type_t::CPU, stream, false));
+            DALI_CALL(daliCopyTensorNTo(
+              &pipeline_handle_,
+              dst,
+              out_id,
+              dataset()->device_type_,
+              dataset()->stream_,
+              false));
           }
           
           *end_of_sequence = false;
```

Comment:
If we use non blocking `daliCopyTensorNTo` I don't know if it is safe to call `daliOutputRelease` later?

Reply:
This is how we do it in [daliop.cc](https://github.com/NVIDIA/DALI/blob/master/dali_tf_plugin/daliop.cc) as well. Do you thing it's also wrong there?

Reply:
It is a good question, but I don't know. Just asking.